tests/cmake/diff_test.sh

```diff
@@ -20,6 +20,8 @@ CMAKE_CURRENT_SOURCE_DIR=$3
 # 4. absolute cmake binary directory
 CMAKE_CURRENT_BINARY_DIR=$4
 
+# 5. when "OFF" just run the tests, otherwise also compare the results
```

Comment:
Nitpicking, but this is not quite true. It compares if set to "ON" and not otherwise.